app/models/fleet.rb

```diff
@@ -22,38 +26,37 @@ def cargo
 
   def available_capacity
     loaded = 0
-    self.cargo.each { |cargo| loaded += cargo.unit.weight * cargo.quantity }
-    (self.capacity * self.quantity) - loaded
+    cargo.each { |cargo| loaded += cargo.unit.weight * cargo.quantity }
+    (capacity * quantity) - loaded
   end
 
   def weight
-    self.quantity * self.unit.weight
+    quantity * unit.weight
   end
 
-  def load_in(carrier,quantity)
-    if self.unit.weight * quantity > carrier.available_capacity
-      quantity = carrier.available_capacity  / self.unit.weight
+  def load_in(carrier, quantity)
+    if unit.weight * quantity > carrier.available_capacity
+      quantity = carrier.available_capacity / unit.weight
       return nil if quantity < 1
     end
     if self.quantity == quantity
-      self.update_attributes(carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
+      update_attributes(carrier: carrier, destination: carrier.destination, arrives_in: carrier.arrives_in)
```

Comment:
I prefer using `update` instead of `update_attributes`, it's shorter.

Reply:
Done. much better.